stack.yaml.lock

```diff
@@ -0,0 +1,19 @@
+# This file was autogenerated by Stack.
+# You should not edit this file by hand.
+# For more information, please see the documentation at:
+#   https://docs.haskellstack.org/en/stable/lock_files
```

Comment:
Should this file be in version control?

Reply:
Yes it should. It's purpose is to give 100% reproducibility on builds so that the next person building the snapshot gets the exact same versions as you got when building it. It's a way to enforce the infamous "it works on my machine".